autocertkit/utils.py

```diff
@@ -891,6 +904,46 @@ def get_equivalent_devices(session, device):
     return ifaces
 
 
+def has_sriov_cap(session, device):
+    master_ref = get_pool_master(session)
+    pifs_ref = get_pifs_by_device(session, device, [master_ref])
+    caps = session.xenapi.PIF.get_capabilities(pifs_ref[0])
```

Comment:
what if the pifs_ref is empty?

Reply:
If any exception occurs in test case then ACK framework will catch it and treat test as failed. This is the ACK Style.